src/collection.js

```diff
@@ -44,21 +65,19 @@ module.exports = function(options) {
   }
 
   var save = function(message, callback) {
-
     createDirectory(function() {
       var promises = Object.getOwnPropertyNames(data)
-        .map(p => write(path.join(root, p + '.json'), JSON.stringify(data[p])))
-
+        .map(p => write(path.join(userroot, p + '.json'), JSON.stringify(data[p])))
       Promise.all(promises).then(function() {
-
-        gitExists(function(exists) {
+        gitExists(userroot, function(exists) {
```

Comment:
Should this be in its own function or at initialisation- as it doesnt make sense to clone the bare when you load the local users repo or save the local users repo, as it would overwrite the changes?
Also the initialisation would mean that the bare repo is empty at the start, so i'm not entirely sure what behaviour you want here